File: mysql/perl/delete/mysql_delete.pl
#! /usr/bin/perl
#
#	mysql/perl/delete/mysql_delete.pl
#
#					May/19/2011
#
# -----------------------------------------------------------------------
use	strict;
use	warnings;
use	utf8;
use	Encode;
#
use DBI;
use lib '/var/www/data_base/common/perl_common';
use sql_manipulate;
use text_manipulate;
# -----------------------------------------------------------------------
print	(encode ('utf-8',"*** 開始 ***\n"));
#
my $id_in = $ARGV[0];
#
print ("$id_in\n");
#
my $constr = 'DBI:mysql:city:host_mysql';
my $user = 'scott';
my $passwd = 'tiger';
my $dbi = DBI->connect($constr, $user, $passwd);
#
sql_manipulate::mysql_utf8_proc ($dbi);
#
sql_manipulate::dbi_delete_proc ($id_in,$dbi);
#
sql_manipulate::sql_show_proc ($dbi);
$dbi->disconnect;
#
print	(encode ('utf-8',"*** 終了 ***\n"));
# -----------------------------------------------------------------------

File: mysql/node/delete/mysql_delete.js
#! /usr/bin/nodejs
// ---------------------------------------------------------------
//	mysql_delete.js
//
//					Dec/24/2012
//
// ---------------------------------------------------------------
var mysql = require('mysql');
var sql_manipulate= require ('/var/www/data_base/common/node_common/sql_manipulate');

// ---------------------------------------------------------------
console.log ("*** 開始 ***");

var id_in = process.argv[2];

console.log (id_in);

var connection = mysql.createClient ({
	host: 'host_dbase',
	user: 'scott',
	password: 'tiger'
	});

connection.query ('Use city');

var command = sql_manipulate.delete_command_gen (id_in);

connection.query(command, function (err, rows)
	{
  	if (err) throw err;
	console.log ("*** 終了 ***");
	});

connection.end();

console.log ("*** ppp ***");
// ---------------------------------------------------------------
